docs/data-structures/queue.mdx

```diff
@@ -0,0 +1,530 @@
+---
+title: Queue
+slug: /data-structures/queue
+---
+
+## Introduction
+
+A ***queue*** is a logical data structure that works on the discipline ***FIFO(First-In-First-Out)***. It is an **ADT (Abstract Data Type)** where the insertion of elements occurs at one end, called **Rear**, and deletion occurs at another end, called **Front**. For example, people standing in a queue to avail some services or cars lined up in a queue are real-world examples of queue data structure.
```

Comment:
No need of unnecessary bolds. Remove them out from: 
- queue
- ADT 
- rear
- front